### DISCUSSION TITLE
LF6 Feedback

### DISCUSSION THREAD

**Discussion by siwelwerd** [TeamBasedInquiryLearning/precalculus#145] [Show and tell]
https://teambasedinquirylearning.github.io/precalculus/2024/LF6.html

Author: @kathypinzon  
Reviewer: @AbbyANoble

**Comment by AbbyANoble** [comment]
[Activity 3.6.2](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-carnival-prices) 

The numbers in the paragraphs (like 128.50 and 4 and 6) aren't in math type.

**Comment by AbbyANoble** [comment]
Answer in (b) is not correct. Correct letter, but wrong equation.

**Comment by AbbyANoble** [comment]
For [Activity 3.6.2(c)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-carnival-prices-4) 
are we expecting them to plug in and check? It might help to have instructions asking that so they aren't trying to figure out how to solve already.

**Comment by AbbyANoble** [comment]
[Activity 3.6.4](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-solution-to-system) 

None of the answer choices are solutions.

**Comment by AbbyANoble** [comment]
Before [Activity 3.6.5](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-system-graph) it might be helpful to have a remark motivating what's coming next. 
Something about how we can check an individual solution, but that's not feasible to actually find a solution. We need methods to solve the system.

**Comment by AbbyANoble** [comment]
[Activity 3.6.7(b)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-system-sub-3) 

Need a space after B: in the answer box.

**Comment by AbbyANoble** [comment]
[Activity 3.6.7(d)](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-system-sub-5) 

Need answer box for (d). It's B!

**Comment by AbbyANoble** [comment]
[Remark 3.6.8](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF6-3-9) 

Typo - refer**r**ed

Might be helpful here to use a numbered list to list the steps to solve. Like

1. Solve one of the equations for one variable.
2. Substitute the expression into the other equation to solve for the remaining variable. 
3. Substitute that value into either equation to find the value of the first variable.

**Comment by AbbyANoble** [comment]
[Activity 3.6.9](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-sub-practice) 

Do we number (letter?) when an activity only has one part?

**Reply by AbbyANoble** [reply]
Same with [Activity 3.6.15](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-practice)

**Reply by kathypinzon** [reply]
I have no idea..... @siwelwerd  how do I not number a task and include an answer?

**Reply by siwelwerd** [reply]
Instead of an `<introduction>` and a `<task>`, just use a single `<statement>`

**Reply by siwelwerd** [reply]
```
<activity>
<statement>
Stuff goes here
</statement>
<answer>
Answer goes here
</answer>
</activity>
```

**Comment by AbbyANoble** [comment]
[Remark 3.6.12](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF6-3-13) 

Typo again in referred. 

Also, maybe a list here like for substitution method.

**Comment by AbbyANoble** [comment]
[Activity 3.6.13](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-elimination-practice) 

Feels like this one needs some scaffolding. The last one had the y's cancel right away. Maybe a lead up with one that's a multiple of the other. Something like "Notice neither of the variables cancel right away if we add the equations together. What could we multiply the first equation by so that they do cancel?" Walk through it, then move to this one with a little bit of scaffolding, leading them to multiply both equations by something to make it cancel.

**Reply by kathypinzon** [reply]
they don't need to multiply since they can just add and the x will cancel (it is just written in the other order)

**Reply by AbbyANoble** [reply]
Ohhhh. Duh. Haha. Then maybe an additional lightly scaffolded problem where they use elimination and need to multiply before they add?

**Comment by AbbyANoble** [comment]
[Activity 3.6.14](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-best-method) and [Activity 3.6.15](https://teambasedinquirylearning.github.io/precalculus/2024/instructor/LF6.html#LF4-practice) 

Needs answers
